py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -139,20 +278,21 @@ def format_data(self, catalogue, spectrographs_data, targetid_spec,
         targetid_spec: int
         Targetid of the objects to format
 
-        forests_by_targetid: dict
-        Dictionary were forests are stored. Its content is modified by this
-        function with the new forests.
-
         reso_from_truth: bool - Default: False
         Specifies whether resolution matrixes are read from truth files (True)
         or directly from data (False)
 
         Return
         ------
+        forests_by_targetid: dict
+        Dictionary were forests are stored. Its content is modified by this
+        function with the new forests.
```

Comment:
docstring forests_by_targetid. The contents are not modified I guess, it just returns. I am also assuming format_data function is just a copy of the old code.

Reply:
You're right, fixed it in a commit. 
Regarding the second part of you comment yes and no. It is part of the old code but not just a copy. It was also part of `read_file` method in `DesiTile` so it was convenient to split it to avoid the duplication